docs/quick_start.md

```diff
@@ -109,9 +146,11 @@ Honeycomb, like most applications, contains a template called `package.json`. Th
     "url": "ccv.brown.edu"
   },
```

Comment:
```suggestion
  },
}
```

Reply:
Add an extra curly brace here to avoid Github formatting things wrong in the preview